Yoshi/Yoshi/Menus/YoshiTableViewMenu/YoshiTableViewMenuCellDataSource.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  YoshiTableViewMenuCellDataSource.swift
+//  Yoshi
+//
+//  Created by Kanglei Fang on 24/02/2017.
+//  Copyright © 2017 Prolific Interactive. All rights reserved.
+//
+
+internal struct YoshiTableViewMenuCellDataSource: YoshiResuableCellDataSource {
+
+    private let title: String
+
+    private let subtitle: String?
+
+    init(title: String, subtitle: String?) {
+        self.title = title
+        self.subtitle = subtitle
+    }
+
+    func cellFor(tableView: UITableView) -> UITableViewCell {
```

Comment:
The cell layout logic is different for every individual cell, the issue before is I accidently include a duplicate data source, now that I removed it, check if there's still logic can be generalized.